shell/src/main/java/alluxio/cli/fs/command/job/JobAttempt.java

```diff
@@ -43,13 +43,13 @@ protected JobAttempt(JobMasterClient client, RetryPolicy retryPolicy) {
    * @return true if an attempt was made, false if attempts ran out
    */
   public boolean run() {
-    if (mRetryPolicy.attempt()) {
+    while (mRetryPolicy.attempt()) {
       mJobId = null;
       try {
         mJobId = mClient.run(getJobConfig());
       } catch (IOException e) {
-        LOG.warn("Failed to start job", e);
-        System.out.println(String.format("Failed to start job with error: %s", e.getMessage()));
```

Comment:
The command output is still needed?

Reply:
@LuQQiu Yeah, maybe we need to add current retry time to the output